runtime/util/openj9_version.c

```diff
@@ -32,9 +32,15 @@ j9print_internal_version(J9PortLibrary *portLib) {
 	PORT_ACCESS_FROM_PORT(portLib);
 
 #if defined(OPENJ9_BUILD)
-	j9tty_err_printf(PORTLIB, "Eclipse OpenJ9 %s %s-bit Server VM (%s) from %s-%s JRE (%s), built on %s %s by %s with %s\n",
+#if defined(J9JDK_EXT_VERSION)
+	j9tty_err_printf(PORTLIB, "Eclipse OpenJ9 %s %s-bit Server VM (%s) from %s-%s JRE (%s) with Extensions for OpenJDK for Eclipse OpenJ9 %s, built on %s %s by %s with %s\n",
```

Comment:
"Extensions for OpenJDK for Eclipse OpenJ9" is still hard coded. Please define this in `closed/openj9_version_info.h.in` and use the define here.

Reply:
If you removed the second insertion of J9VERSION_STRING that would also be appreciated, although unrelated to this change. There is no reason to include the exact same version twice.